include/silo_api/info_handler.h

```diff
@@ -1,26 +1,25 @@
 #ifndef SILO_INFOHANDLER_H
 #define SILO_INFOHANDLER_H
 
-#include <Poco/Net/HTTPRequestHandler.h>
 #include <Poco/Net/HTTPServerRequest.h>
 #include <Poco/Net/HTTPServerResponse.h>
 
+#include "silo_api/request_handler.h"
```

Comment:
We could make this a forward declaration `class RestResource;`. This makes the build faster, and it is clearer, where `RestResource` comes from.

Reply:
I didn't manage to make it work since `InfoHandler` inherits from `RestResource`, i.e. The this header file needs to know about the structure of `RestResource`.

Reply:
Sadly, me neither... 
I've pushed a commit, where each handler is put in its own file. Thus, `RestResource` can be found more easily. We can also think about putting the request handlers in their own folder. What do you think?

Reply:
Sounds reasonable, but for now IMO the `/silo_api` is already the separate folder. If the api layer grows further, we can still split it.